tianshou/highlevel/env.py

```diff
@@ -69,17 +68,24 @@ class VectorEnvType(Enum):
     """Parallelization based on `subprocess`"""
     SUBPROC_SHARED_MEM = "shmem"
     """Parallelization based on `subprocess` with shared memory"""
+    SUBPROC_SHARED_MEM_FORK_CONTEXT = "shmem_fork"
+    """Parallelization based on `subprocess` with shared memory and fork context (relevant for macOS)"""
```

Comment:
If you have a link to some docs, might be good to add it :). Or is the info trivial to google?

Reply:
Also https://github.com/Farama-Foundation/Gymnasium/issues/222

Reply:
Then worth mentioning. It's nice if we can save a user (or future us) from going down a longish web-search to understand when and why these things would need to be adjusted